src/implementations/spaces/multi_continuous_space.jl

```diff
@@ -21,12 +21,12 @@ Similar to [`ContinuousSpace`](@ref), but scaled to multi-dimension.
 """
 MultiContinuousSpace(low, high) = MultiContinuousSpace(promote(low, high)...)
 
-Base.eltype(::MultiContinuousSpace{T}) where {T} = T
+Base.eltype(::MultiContinuousSpace{T}) where {T} = eltype(T)
```

Comment:
Ah, sorry I changed the meaning of `eltype` some time ago. I prefer that `eltype` returns the type of an element in the `AbstractSpace`. Here the element of a `MultiContinuousSpace` is an `AbstractArray`. What do you think?

Reply:
Makes sense.